client/src/GlobalStyles.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+import Roboto from "./fonts/roboto-regular.ttf";
+import Lora from "./fonts/lora-mediumItalic.ttf";
+
+const GlobalStyles = () => {
+  return (
+    <Global
+      styles={css`
+        *,
+        *:before,
+        *:after {
+          box-sizing: border-box;
+          margin: 0;
+        }
+        :root {
+          height: 100vh;
+          width: 100vw;
+          --font-color-dark: #383636;
+          --font-color-light: #f5f6fa;
+          --base-bg-color: linear-gradient(
+            to top,
+            #d5d4d0 0%,
+            #d5d4d0 1%,
+            #eeeeec 31%,
+            #efeeec 75%,
+            #e9e9e7 100%
+          );
+          --menu-bg-color: #cebebe;
+          --header-bg-color: #a79292;
+          --bubble-bg-color: #cebebe;
+        }
```

Comment:
I would suggest splitting variables and stylings into `:root` and `html`.
See https://github.com/slawoe/moodster/pull/40#discussion_r488766511

Reply:
Thanks, I separated variables and rules.